music_rnn/src/model.ts

```diff
@@ -137,10 +137,11 @@ export class MusicRNN<T extends magenta.controls.ControlSignalUserArgs> {
    * @param steps How many steps to continue.
    * @param temperature The softmax temperature to use when sampling from the
```

Comment:
While you're at it, please add the (Optional) prefix to temperature.

Reply:
Done.